examples/network/layout/hierarchicalLayoutDynamicChanges.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Vis Network | Layouts | Dynamic Hierarchical Layout Changes</title>
```

Comment:
I'm not sure if this PR is the best place to introduce a completely new example, but I don't mind!

Reply:
This isn't exactly new. It slipped here from #115.